platform/ios/src/MGLMapView.mm

```diff
@@ -2707,6 +2712,19 @@ - (void)_setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count
     [self didChangeValueForKey:@"visibleCoordinateBounds"];
 }
 
+- (std::vector<mbgl::LatLng>)normalizeCoordinatesForAntimeridianSW:(CLLocationCoordinate2D)sw ne:(CLLocationCoordinate2D)ne
+{
+    CLLocationDegrees longitude = (180 + (180 - fabs(sw.longitude))) * -1;
```

Comment:
If I’m not mistaken, the `mbgl::LatLngBounds` class can handle spanning the antimeridian – that’s how the existing shape annotation functionality is able to handle the antimeridian gracefully. See the `mbgl::LatLngBounds::extend()` method in particular.